### PR TITLE
Add a Contributing.md

### LINKED ISSUE #78: Create some boilerplate text about pull requests to CVE entries
Many people seem to enjoy editing CVE entries directly to demonstrate what they'd like to see changed -- this is especially true for people who disagree with ADP opinions on CPEs, CWEs, and CVSS scores. And, while we definitely want to hear about these other opinions, PRs tend to see this kind of workflow:

- External contributor forks and edits and pull requests, all in the usual way
- Maintainer sees the PR, might have a little discussion, and possibly agrees with the external assessment.
- If agreed, maintainer might land the PR, or might not, because the actual change needs to happen to the relevant metric on the backend, and publish *that* fix.
- If the provided PR doesn't land, the contributor doesn't get the nice little dopamine hit from having a successfully merged PR.
- If the provided PR does land, it ends up being kind of meaningless in a practical sense, since it'll be overwritten by the next scheduled push from the backend. So, at best, this is a NOP workflow. At worst, merge conflicts happen.

The easy answer is to reject all PRs to CVEs, but that kind of sends a message of "thanks, don't care."

Perhaps some guidance in a CONTRIBUTING.md doc about how to use PRs? Basically, we should outline that we definitely can accept PRs to pretty much anything that's **not** a CVE entry, and cannot practically accept PRs to CVEs.(but can perform edits and push) 

Up for discussion. I don't want to rob people of internet points, but I also don't want to encourage broken and risky workflows.

### PR DESCRIPTION
**Pull request by todb-cisa**
This PR fixes #78, though the job of cleaning up the boilerplate text for new issues and PRs is left to be done.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Pull Requests
+
+Patches accepted! Please note, though, that there's a bit of a wrinkle with submitting PRs against CVE JSON files
+in Vulnrichment repository. Because these files are generated through a backend process here at CISA, your changes
+are quite likely to be overwritten the next time the corpus of enriched CVEs are generated, and your specific
+PR'ed commit may be lost. However, we will make every effort to merge your PR in the normal GitHub way, so even in that
+case, your contribution will be recognized by GitHub and your account will be listed as a contributor. Occasionally,
+though, such merges can create conflicts which we'd rather avoid. This is especially true if you are submitting a PR
+that affects a large number of CVEs. In these cases, the PR is still helpful to see what you're trying to accomplish,
+but the ultimate change may be a systemic change on our backend processing, and your origianl PR may be closed without
```
Comment (jwoytek-cisa) [inline CONTRIBUTING.md]: Spelling: original

**Review by jwoytek-cisa — CHANGES_REQUESTED** [review]
Looks good! One minor spelling issue identified.

File: CONTRIBUTING.md (hunk 1 above)
Comment (todb-cisa) [reply CONTRIBUTING.md]: Fixed! Someday I'll figure out the real GitHub workflow of fixing things directly from the PR. I end up always having to open a new window, navigate to the branch (or do it locally on the CLI), and fix it "outside" of the PR. Surely there's a better way.

**Comment by todb-cisa** [comment]
Man I wish I knew how to get rid of this "change requested" thing without being rude about it. Someday I'll learn GitHub.

**Review by jwoytek-cisa — APPROVED** [review]

**Comment by jwoytek-cisa** [comment]
> Man I wish I knew how to get rid of this "change requested" thing without being rude about it. Someday I'll learn GitHub.

Yeah I'm often trying to figure out the "right way!" I marked the changes as approved, which appears to be the right way for this one!

**Pull request merged**